wiki/en/FAQ.md

```diff
@@ -57,7 +57,7 @@ No. If you just intend to connect to other people’s servers, then all you need
 <img src="{% include img/en-screenshots/diagram-overview.inc %}" loading="lazy" alt="Diagram showing how Jamulus works">
 
 
-Jamulus works on the client-server principle. Everybody’s audio is sent to a server, mixed and processed there. Afterwards, the audio is sent back to each client. If a server is made public and registered on a public directory, its information will be broadcast to all clients.
+Jamulus works on the client-server principle. Everybody’s audio is sent to a server, mixed and processed there. Afterwards, the audio is sent back to each client. If a server is registered in a Jamulus directory, the server's information will be broadcast to all clients using that directory.
```

Comment:
Hm.  I don't like "broadcast" here now I see it.  The client _asks_ the directory for the information.  Each client gets a _separate_ response from the directory, it's not a single broadcast signal.